src/main/java/fr/jcgay/maven/plugin/buildplan/AbstractLifecycleMojo.java

```diff
@@ -34,6 +40,10 @@
     /** Allow to specify which tasks will be used to calculate execution plan. */
     @Parameter(property = "buildplan.tasks", defaultValue = "deploy")
     private String[] tasks;
+    
+    /** Allow to specify an output file to bypass console output */
+    @Parameter(property = "buildplan.outputFile")
+    private String outputFile;
```

Comment:
Switched to `File`. But it is creating an output file in each projects root on a reactor. I haven't done much maven plugin development. Should I get the execution root and reuse that directory for the file? Then add a header to preceding the output for the name of the module?

https://github.com/apache/maven/blob/maven-3.5.0/maven-core/src/main/java/org/apache/maven/execution/MavenSession.java#L148